doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -2777,25 +2779,31 @@ XROAD_PROXY_UI_API_PARAM
 XROAD_SIGNER_CONSOLE_PARAM
 ```
 
-## 22 Configuring account lockout
+## 22 Additional Security Hardening
 
-You may want to improve the security of your X-Road instance by configuring an account lockout policy in your Security Server Admin UI authentication. When trying to log in to the Security Server Admin UI with a locked account, the login screen will display a generic login error without disclosing the reason or any other login information.
+You may want to harden the security of your X-Road instance by configuring additional security policies in your Security Servers. The security measures that are introduced in this guide are common security policies that can be configured on operating system level.
 
-X-Road uses the Linux Pluggable Authentication Modules (PAM) to authenticate users. This makes it easy to configure the account management to your liking. The service to configure the account lockout to is `xroad`.
+X-Road uses the Linux Pluggable Authentication Modules (PAM) to authenticate users. This makes it easy to configure the account management to your liking. The example PAM configurations provided in this guide may or may not work on your system depending on your system and existing PAM configurations. Note that editing the PAM configurations will take effect immediately without the need to restart anything.
 
-For configuring the account lockout for the X-Road Security Server Admin UI in production, please refer to [The Linux-PAM System Administrator's Guide](http://www.linux-pam.org/Linux-PAM-html/Linux-PAM_SAG.html) for the full documentation on how to configure PAM.
 
-### 22.1 Considerations and risks
+For configuring the following security policies for the X-Road Security Server Admin UI in production, please refer to [The Linux-PAM System Administrator's Guide](http://www.linux-pam.org/Linux-PAM-html/Linux-PAM_SAG.html) for the full documentation on how to configure PAM.
+
+### 22.1 Configuring account lockout
+
+Configuring an account lockout policy in your Security Server Admin UI authentication will provide an extra layer of defence against password guessing attacks. After configuring the account lockout, when trying to log in to the Security Server Admin UI with a locked account, the login screen will display a generic login error without disclosing the reason or any other login information.
```

Comment:
Configuring an account lockout policy in your Security Server Admin UI authentication will provide an extra layer of defense against brute force, or password guessing attacks.

Reply:
I was thinking that `password guessing attacks` would already cover attacks like brute force and password spraying. But it is a good clarification so I updated the PR 👍

Done in 684f435

Reply:
Thank you, Caro.  I wasn't sure about the name of the attack either, but I thought it was always better to ask than to doubt.